frontend/packages/container-security/src/components/image-manifest-vuln.tsx

```diff
@@ -314,7 +318,10 @@ export const ImageManifestVulnList: React.FC<ImageManifestVulnListProps> = (prop
 export const ImageManifestVulnPage: React.FC<ImageManifestVulnPageProps> = (props) => {
   const { t } = useTranslation();
   const { showTitle = true, hideNameLabelFilters = true } = props;
-  const namespace = props.namespace || props.match.params.ns || props.match.params.name;
+  const namespace =
+    props.namespace || (props.match && props.match.params)
+      ? props.match.params.ns || props.match.params.name
+      : null;
```

Comment:
the page fails to load when I am in a namespace and go to search ->  `IMV`

Can you try doing :
```suggestion
const namespace = props.namespace ?? props.match?.params?.ns ?? props.match?.params?.name ?? null;
```

Reply:
Good catch, thanks. Replaced it with just

```tsx
  const namespace = props.namespace ?? props.match?.params?.ns ?? props.match?.params?.name;
```

I tried the optional chaining before but it doesn't work. Now it works fine, thanks.